File: Baekjoon/구현/3003.py
# https://www.acmicpc.net/problem/3003

import sys
input = sys.stdin.readline

chess = list(map(int, input().split()))
standard = [1, 1, 2, 2, 2, 8]

for i in range(6):
    print(standard[i] - chess[i], end=' ')

File: Baekjoon/구현/3003.cpp
// https://www.acmicpc.net/problem/3003

#include <iostream>
#include <vector>
#include <cmath>
using namespace std;

int main() {
    vector<int> pieces = {1, 1, 2, 2, 2, 8};
    for (int i = 0; i < 6; i++) {
        int n;
        cin >> n;
        cout << pieces[i] - n << ' ';
    }
    return 0;
}